### PR TITLE
Update configuration.md to describe new oidc-registration flag

### PR DESCRIPTION
**Pull request by lectrician1**

### CODE REVIEW

File: technical-guide/configuration.md [hunk 1]
```diff
@@ -224,6 +224,16 @@ endpoint.
...
+<br />
+
+__Since version 2.1.2__
+
+Allows users to register and login with oidc without having to previously 
+register with another method.
+
+```bash
+PENPOT_FLAGS="[...] oidc-registration"
+```
```
Comment (superalex) [inline technical-guide/configuration.md:236]: should be enable-oidc-registration

**Comment by superalex** [comment]
Thanks @lectrician1 

Could you please squash your two commits in just one?

**Comment by lectrician1** [comment]
hhh ya now i have to clone the whole repo tho :(

**Comment by lectrician1** [comment]
eh codespaces worked :)

**Comment by superalex** [comment]
Thank you very much! ;)

**Pull request merged**